app/models/jwt_api_entreprise.rb

```diff
@@ -3,6 +3,8 @@ class JwtApiEntreprise < ApplicationRecord
   has_many :contacts, dependent: :delete_all
   has_and_belongs_to_many :roles
 
+  scope :unexpired, -> { where('exp >= ?', Time.zone.now.to_i) }
```

Comment:
Comme l'expiration commence à partir de cette période, je propose de changer en supérieur strict :

```suggestion
  scope :unexpired, -> { where('exp > ?', Time.zone.now.to_i) }
```

Reply:
c'est kifkif, on parle de Timestamp à la seconde d'après c'est résolu

Reply:
Si c'est dans un commit à part moi ça me choque pas.

Reply:
Effectivement dans la spec JWT `exp` c'est bien le temps *à partir* duquel le jeton n'et plus valide. 

Après comme le dis @Haelle à la seconde près on aurait pas eu beaucoup d'effet de bords... Clairement si c'était le seul changement demandé j'aurais passé 😂

Reply:
J'aime bien respecter à la lettre les spécifications. :)